sdk/core/azure-core/inc/azure/core/credentials/credentials.hpp

```diff
@@ -60,22 +60,38 @@ namespace Azure { namespace Core { namespace Credentials {
      *
      * @param tokenRequestContext #Azure::Core::Credentials::TokenRequestContext to get the token
      * in.
-     * @param context #Azure::Core::Context so that operation can be cancelled.
+     * @param context A context to control the request lifetime.
```

Comment:
In this context (no pun intended), for getting an access token, is there a request being sent whose lifetime is actually being controlled by the context, or does it server a different purpose?

Reply:
Yes, the request is being sent. Credential is client. Other clients give you files (storage) or secrets (keyvault). Credential classes retreive auth tokens from the server. This is an oversimplification, to an extent.